CHANGELOG.md

```diff
@@ -1,11 +1,12 @@
 # Changelog
 
-## 3.11.3 - Released 25 Apr 2025
+## 3.11.4 - Unreleased
+
+- [Accessibility] Added configurable background color to time and checkmark in messages. `parley_meta_background_corner_radius` `parley_meta_background_color`
+
```

Comment:
Add that `parley_message_time_background_color` is renamed to `parley_meta_background_color` and same for 

parley_user_image_time_background  and parley_agent_image_time_background